unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -782,6 +782,10 @@ public IEnumerator checkInitializeAgentLocationAction()
             }
         }
 
+        public void MoveInDirection(Vector3 direction) {
```

Comment:
Perhaps this is a duplicate of MoveRelative?
https://github.com/allenai/ai2thor/blob/master/unity/Assets/Scripts/BaseFPSAgentController.cs#L1713

Moreover, allowing one to specify a non-0 `y` in `direction kinda scares me. I'm not actually sure what it would do. For better or worse, all actions in the BaseFPSAgentController assume that the agent can only move along the XZ axes.

Reply:
Move relative is in the agent's coordinate frame, whereas move in direction is using the global coordinate frame. Happy to refactor to only allow x/z movement.

Reply:
Oh, I see. Seems useful then.

Yeah, I think restricting to x/z would work best.

Reply:
The other note: I would probably rename this to something like `MoveGlobal` to make the difference between `MoveRelative` clear.

Alternatively, since `MoveRelative` is undocumented (and thus easy to deprecate), it could just be worth adding a `coordinateSpace: string` parameter to `MoveInDirection()`, which allows for either specifying `relative` or `world`.

Reply:
👍 I have changed it `MoveGlobal` (note however that the `MoveRelative` command is a bit weird because it uses the usual video game idea of movement with very few checks as opposed to what we do). I also updated the DebugInputField.cs script which might be worth looking at.